### ISSUE TITLE
Weird background with rose-pine

### ISSUE THREAD

**Issue by razprod** [ThePrimeagen/init.lua#37]
I followed the youtube tutor and also tried to simply clone this repo but results are the same:
![nvim — nvim   ▸ node — 109×31 2023-01-19 17-13-13](https://user-images.githubusercontent.com/87066774/213479682-8a23460a-eb15-4d0c-855c-7c5a941dc7ce.png)

I cannot seem to figure out why this is happening...
![nvim — nvim   ▸ lua-language-server — 109×31 2023-01-19 17-14-29](https://user-images.githubusercontent.com/87066774/213479862-3e51d694-b66d-43c4-8e39-bee7f001a61f.png)

Please anyone...
@raz-cohen

**Comment by bryanboateng** [comment]
My guess is that your Terminal doesn't support 24-Bit color. Which one are you using?

**Comment by razprod** [comment]
> My guess is that your Terminal doesn't support 24-Bit color. Which one are you using?

Tnx for the reply,
I'm on a macOS 13-inch, 2019 and using the default terminal...
I've a wsl installed on my windows and I've successfully managed to make it work also with the colors..

**Comment by bryanboateng** [comment]
Yes, sadly the default macOS terminal doesn't support 24-bit colors per default.

**Comment by bryanboateng** [comment]
Could you please close the issue if the problem is solved?

**Comment by raz-cohen** [comment]
> Could you please close the issue if the problem is solved?

I’m not able to..
